int-config.toml

```
@@ -1,58 +1,58 @@
 [cveDict]
-  Type = "sqlite3"
-  SQLite3Path = "/data/vulsctl/docker/cve.sqlite3"
+Type = "sqlite3"
```

In the server section below, the property names seem to start with a lowercase letter. Can we unify them one way or the other?
```suggestion
type = "sqlite3"
```

---

Good catch! fixed!